app/static/src/app/components/basic/BasicApp.vue

```diff
@@ -11,7 +11,7 @@
       </wodin-tabs>
     </template>
     <template v-slot:right>
-      <wodin-tabs id="right-tabs" :tabNames="['Run', 'Sensitivity']">
+      <wodin-tabs id="right-tabs" :tabNames="['Run', 'Sensitivity']" @tabSelected="rightTabSelected">
```

Comment:
Since selected tab is added to the state, you shouldn't need to emit.

Reply:
I need to emit it from WodinTabs, because there are two instances of WodinTabs (left and right), and we only need to commit when the right-hand (aka visualisation) tab changes - so I think it needs to be the parent of those tabs, which knows which is which, which commits the mutation to the store.